src/main.zig

```diff
@@ -1010,6 +1010,14 @@ fn onCompletion(io: *ourio.Ring, task: ourio.Task) anyerror!void {
                         );
                         return;
                     },
+                    error.AccessDenied => {
+                        std.debug.print("cannot access '{s}': Permission denied\n", .{task.req.open.path});
```

Comment:
is this how you want to print? i saw you're using a writer to stderr but i couldn't figure out how to pass it to this callback